plugins/module_utils/dnac.py

```diff
@@ -992,6 +993,182 @@ def check_tasks_response_status(self, response, api_name):
 
         return self
 
+    def update_result(self, status, changed, msg, log_level, data=None):
```

Comment:
Can you give any other name?
record_operation_result or set_operation_result

For better clarity:
status - operation_status, 
changed - is_changed, 
msg - status_message, 
data - additional_info